pulpcore/tests/functional/__init__.py

```diff
@@ -332,3 +344,17 @@ def _random_artifact_factory():
         return gen_object_with_cleanup(artifacts_api_client, temp_file)
 
     return _random_artifact_factory
+
+
+@pytest.fixture
+def add_to_disc_cleanup(cli_client):
```

Comment:
```suggestion
def add_to_filesystem_cleanup(cli_client):
```

This reads a bit clearer to me (this is totally subjective).